### PR TITLE
Add circle.yml for publishing

### PR DESCRIPTION
**Pull request by phubars**

### CODE REVIEW

File: .circle/config.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      - restore_cache:
+          key: yarn-{{ .Branch }}-{{ checksum "yarn.lock" }}
+
+      - run:
+          name: Install dependencies
+          command: |
+            echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" >> .npmrc
+            yarn
+
+      - run:
```
Comment (arunsivasankaran) [inline .circle/config.yml:24]: do we still need green keeper step? we dont use it anymore

File: .circle/config.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+          paths:
+            - "~/.yarn-cache"
+
+      - store_artifacts:
+          path: coverage/
+
+      - store_test_results:
+          path: test-output/
+
+      - run:
```
Comment (arunsivasankaran) [inline .circle/config.yml]: ditto

**Review by arunsivasankaran — DISMISSED** [review]

File: .circle/config.yml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+          name: Maybe greenkeeper update
+          command: |
+            if [[ "${CIRCLE_BRANCH}" =~ greenkeeper* ]]; then
+                greenkeeper-lockfile-update
+            fi
+
+      - run:
+          name: Run tests
+          command: |
+            yarn test:unit
```
Comment (arunsivasankaran) [inline .circle/config.yml]: I don't see this command in scripts?

File: .circle/config.yml (hunk 1 above)
Comment (phubars) [reply .circle/config.yml:24]: chu right. I copy pasta'd

**Review by arunsivasankaran — APPROVED** [review]

**Pull request merged**